addons/mail/static/src/core/common/message.xml

```diff
@@ -36,7 +36,7 @@
                                 <strong class="me-1"><t t-if="message.author" t-esc="message.author.name"/><t t-else="" t-esc="message.email_from"/></strong>
                             </span>
                             <t t-if="!isAlignedRight" t-call="mail.Message.notification"/>
-                            <small t-if="!message.is_transient" class="o-mail-Message-date text-muted opacity-75" t-att-class="{ 'me-2': !isAlignedRight }" t-att-title="message.datetimeShort">
+                            <small t-if="!message.is_transient" class="o-mail-Message-date text-muted opacity-75" t-att-title="message.datetimeShort">
```

Comment:
I still don't know why you remove that, to me, it does not seem related to the issue to fix. 🤔 

May I miss something?

Reply:
Hello @xavierbol 

Since we are adding a space in every scenario I think we should remove it to avoid the extra space. If i miss something please let me know 😃

Reply:
No, I don't agree, it is not related to the issue we try to fix.